client/templates/initiatives/initiatives.js

```diff
@@ -7,9 +7,9 @@ Template.initiatives.helpers({
       }), cursor);
```

Comment:
When using the search box, there is no logic applied for only retrieving active Initiatives. We should apply that same logic to it.